content/hi/reliability.md

```diff
@@ -0,0 +1,7 @@
+---
+title: विश्वसनीयता (Reliability)
+status: Completed
+category: गुण
+---
```

Comment:
We need to add the 'tags' column too. 
Ref: https://github.com/cncf/glossary/blob/main/content/en/reliability.md

Reply:
Updated :)